README.md

```diff
@@ -356,6 +356,28 @@ Source code and an Eclipse project for the Flexy Java app are made available in
 #### Development Environment
 The Flexy Java application component was developed using a standard Ewon Java development environment. Documentation and additional information about the Ewon Java development environment is available in the Ewon Java Toolkit User Guide \(J2SE\) at [https://developer.ewon.biz/content/java-0#dev-documents](https://developer.ewon.biz/content/java-0#dev-documents).
 
+### Cloning
+
+Using the git client of your choice, clone the https://github.com/hms-networks/flexy-thingworx-connector repository.
+
+Using HTTPS:
+```console
+> git clone https://github.com/hms-networks/flexy-thingworx-connector.git --recursive
+```
+
+Using SSH:
+```console
+> git clone git@github.com:hms-networks/flexy-thingworx-connector.git --recursive
+```
+
+### Libraries
```

Comment:
should this be named Submodules rather than Libraries?

Reply:
I think it should include both, so I was planning on saying "Libraries (Git Submodules)". I've heard them referred to as libraries, so I think it might be confusing to not include that term.